packages/transport-circuit-relay-v2/src/server/index.ts

```diff
@@ -93,9 +100,19 @@ export interface RelayServerEvents {
   'relay:advert:error': CustomEvent<Error>
 }
 
-const defaults = {
-  maxOutboundStopStreams: MAX_CONNECTIONS
-}
+const configValidator = object({
+  hopTimeout: number().integer().min(0).default(DEFAULT_HOP_TIMEOUT),
+  reservations: object({
+    maxReservations: number().integer().min(0).default(DEFAULT_MAX_RESERVATION_STORE_SIZE),
+    reservationClearInterval: number().integer().min(0).default(DEFAULT_MAX_RESERVATION_CLEAR_INTERVAL),
+    applyDefaultLimit: boolean().default(false),
```

Comment:
I think this needs to default to true, otherwise all js nodes will be open relays by default.

Reply:
Can we not reuse the config validator from the reservation store here?